GitSocial/GitSocial/GSTextField.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  GSTextField.swift
+//  GitSocial
+//
+//  Created by Dillon on 11/27/21.
+//
+
+import UIKit
+
+class GSTextField: UITextField {
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configure()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    /// Initialize new GSTextField with a placeholder text. The `configure()` method called
+    /// by this init sets 'translatesAutoresizingMaskIntoConstraints' to false, so that you do not
+    /// need to set this property at the call site.
+    init(placeholder: String = "") {
+        super.init(frame: .zero)
+        self.placeholder = placeholder
+        configure()
+    }
+    
+    private func configure() {
+        layer.cornerRadius = 10
```

Comment:
Let's try to keep configuration methods consistent and predictable.
```suggestion
        translatesAutoresizingMaskIntoConstraints = false
        layer.cornerRadius = 10
```

Reply:
Refer to #3